newapi/tests/integ/test_torsf_stats.py

```diff
@@ -0,0 +1,47 @@
+from urllib.parse import urlencode
+
+import pytest
+
+
+def api(client, subpath, **kw):
+    url = f"/api/v1/{subpath}"
+    if kw:
+        assert "?" not in url
+        url += "?" + urlencode(kw)
+
+    response = client.get(url)
+    assert response.status_code == 200
+    assert response.is_json
+    return response.json
+
+
+@pytest.mark.skipif(not pytest.proddb, reason="use --proddb to run")
+def test_torsf_stats_cc(client, log):
+    url = "torsf_stats?probe_cc=IT&since=2021-07-28&until=2021-07-29"
```

Comment:
Do we have some metrics where `anomaly_count` is > 0? If so it might be worth adding a sample check for that too.